power on terminals after data registration (#451)
SUSE/doc-susemanager: modules/retail/pages/retail-admin.adoc

. In the {susemgr} {webui}, navigate to menu:Main Navigation[Systems > System Groups], and click the btn:[Create Group] button.
. In the [guimenu]``Create System Group`` dialog, create a new system group, using the hardware type group name you created earlier.

> Now, all the information about your point-of-service terminals is prepared.  Power on your point-of-service terminals and the branch server will bootstrap the terminals according to the provided data.

Reviewer: ```suggestion
Power on your point-of-service terminals.
The branch server will bootstrap the terminals according to the data you have provided.
```
